lib/rules/named-grid-areas-no-invalid/__tests__/index.js

```diff
@@ -103,8 +112,8 @@ testRule({
 		},
 		{
 			code: stripIndent`
-				a { grid-template-areas: 
-			   	""; 
+				a { grid-template-areas:
+			   	"";
```

Comment:
[suggestion] The whitespaces for indentation seem to be wrong.

```suggestion
					"";
```